.github/workflows/build-webapi.yaml

```diff
@@ -74,3 +74,38 @@ jobs:
         run: |
           wget -t 5 --waitretry=5 ${DATA_ENDPOINT}
 
+  api-comparison:
+      needs: container-dotnet-build
+      runs-on: ubuntu-latest
+      defaults:
+        run:
+          working-directory: ./src/CarbonAware.WebApi/src
+      container:
+        image: mcr.microsoft.com/dotnet/sdk
```

Comment:
nit: Does it make sense to pin the version here? `mcr.microsoft.com/dotnet/sdk:6.0`